packages/clients/textLocator/src/plugins/GeometrySearch/components/Tree.vue

```diff
@@ -12,7 +12,6 @@
         </template>
         {{ item.name }}
         <template v-if="item.type === 'toponym' && item.feature">
-          &nbsp;
```

Comment:
It was previously used for spacing. I later styled the element so that a margin is there in the AddressSearch. Little did we realize that now the content of the GeometrySearch were overly spaced. But now I noticed and removed it.